mkosi.images/system/mkosi.conf.d/10-debian-ubuntu/mkosi.conf

```diff
@@ -26,8 +26,8 @@ Environment=
                           udev"
 
 Packages=
-        ^libtss2-esys-[0-9.]+-0$
-        ^libtss2-mu-[0-9.]+-0$
+        ^libtss2-esys-[0-9.-t]+$
+        ^libtss2-mu[0-9.-t]+$
```

Comment:
Is the position of `-` before `t` intended? What string is expected to be matched?

Reply:
The same as before except optionally suffixed with t64. But the current approach doesn't work because both the library with and without the suffic are in the ubuntu repos and both are always selected.